Gems/PythonAssetBuilder/Editor/Scripts/scene_api/scene_data.py

```diff
@@ -272,5 +290,242 @@ def mesh_group_add_tangent_rule(self, mesh_group: dict, tangent_space: int, tspa
 
         mesh_group['rules']['rules'].append(rule)
 
+    def __add_physx_base_mesh_group(self, name: str, physics_material: typing.Optional[str]) -> dict:
```

Comment:
do you think in the future this code will be broken out into a separate module?

Reply:
Based on your update, this should probably all be updated at some point to match the same style, so yes :)